src/process_utility.c

```diff
@@ -1252,7 +1439,12 @@ process_index_end(Node *parsetree, CollectedCommand *cmd)
 	CatalogSecurityContext sec_ctx;
 	Cache *hcache;
 	Hypertable *ht;
-	bool handled = false;
+	CreateIndexInfo info = {
+		.stmt = stmt,
+		.extended_options = hypertable_index_options,
+	};
+
+	hypertable_index_options = hypertable_index_options_default;
```

Comment:
We shouldn't clobber the variable during the end function because of possible errors before we get here.